src/App.tsx

```diff
@@ -7,6 +7,7 @@ import { Header } from './components/Header';
 export const App: React.FC = () => {
   return (
     <FavouriteProvider>
+      <div className="wrapper">
```

Comment:
Try to avoid wrapper, cause we want to have `<Header>` `<main>` and `<Footer>` on top of elements root.
Maybe just remove all styles from wrapper to `body`?